javascript/javascript1/week1/ez-namey/ez-namey.js

```diff
@@ -0,0 +1,36 @@
+
+const firstWords = [
+    "Dynamic",
+    "Transparent",
+    "Inovative",
+    "Sustainable",
+    "Customer-Centric",
+    "Collaborative",
+    "Adaptable",
+    "Data-Driven",
+    "Empatthetic",
+    "Scalable"
+];
+const secondWords = [
+    "Evolving",
+    "Balancing Positivity & Realism",
+    "Ethical",
+    "Empowering",
+    "Tech-savvy",
+    "Global",
+    "Resilient",
+    "User-Friendly",
+    "Secure",
+    "Reliable",
+    "Fast-Paced"
+];
+
+const startupName = "NouGol"; // First letters of my first and last name ;)
+
+
+const randomFirst = Math.floor(Math.random() * 10);
+const randomSecond = Math.floor(Math.random() * 10);
+
+
+console.log("The startup: " + startupName + " contains " + (startupName.length) + " characters.");
```

Comment:
Here the objective is to console.log the randomly generated start up name using the 2 arrays you created (firstWords & secondWords).